src/main/webapp/index.html

```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html>
 <meta charset="utf-8" />
-
+<script src="script.js" defer></script>
 <title>ZoomTube</title>
 <link rel="stylesheet" href="style.css" />
 <script src="script.js"></script>
```

Comment:
Script already included above.  Maybe make this the deferred one and delete the other? kinda weird having it before `title`

Reply:
missed it, thanks